spec/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../book'
+
+book = Book.new('Ruby', 'Promise')
+
+describe '#Check the Book Class' do
+  it 'returns the correct title' do
+    expect(book.title).to eql 'Ruby'
+  end
+
+  it 'returns the correct author' do
+    expect(book.author).to eql 'Promise'
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods of all the classes, Kindly make sure the `add_rental`  method has (at least) an 'it' block 👇🏻 check your book.rb file

```ruby
  def add_rental(rental)
    @rentals.push(rental)
    rental.book = self
  end
```